Online/DebugOverlay.cs

```diff
@@ -152,35 +197,37 @@ public static void Update(RainWorldGame self, float dt)
                 incomingLabels.Clear();
 
                 int line = 0;
-                foreach (OnlinePlayer player in OnlineManager.players)
+                foreach (playerCache.Individual idv in playersWritten.items)
                 {
-                    if (player.statesWritten || player.eventsWritten)
-                    {
-                        FLabel label = new FLabel(Custom.GetFont(), player.ToString()) { alignment = FLabelAlignment.Left, x = 5.01f, y = screenSize.y - 25 - 15 * line };
-                        if (player.eventsWritten)
-                        {
-                            label.color = new Color(1, 0.5f, 0);
-                        }
-                        overlayContainer.AddChild(label);
-                        outgoingLabels.Add(label);
-                        line++;
-                    }
+                    var player = idv.player;
+                    var playerTruePing = Math.Max(1, player.ping - 16);
```

Comment:
can OnlineManager's rawUpdate fire more than 60 times p second? If so, this hardcoded var should be changed...